src/components/ToDoContainer.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect} from 'react';
+import AddToDoForm from './addToDoForm';
+import ToDoList from './todolist';
+import style from './ToDoList.module.css'
+
+function ToDoContainer ({tableID , tableName}) {
+  
+   //state and basic API url
+   const [toDoList, setToDoList]= React.useState([]); 
+   const [isLoading, setIsLoading]= React.useState(true);
+   const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${tableID}?maxRecords=10&view=Grid%20view`
+   const updateURL = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${tableID}/`
+   
+   //read the list from local storage after load 
+   useEffect(()=> {
+     //fetching data from AirTable and parsing it
+    fetch(url, 
+     {method: "GET",
+     headers: {Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`}})
```

Comment:
This is difficult to read.  You should use indentation inside the brackets and such to make it easier to read:

```
.then((result)=> {
    return result.json();
})
```